Makefile

```diff
@@ -38,8 +37,18 @@ install-bin: bin/kind bin/kubectl-kuttl
 create-kind-cluster: $(KUBECONFIG)
 
 $(KUBECONFIG): install-bin
-	bin/kind create cluster --wait 10s --config=test/kind/$(KUBERNETES_VERSION).yaml
+	git clone --depth 1 https://github.com/mesosphere/kubeaddons-enterprise-tests.git --branch master --single-branch
+	bin/kind create cluster --wait 10s --image=kindest/node:v$(KUBERNETES_VERSION)
 
 .PHONY: kind-test
 kind-test: create-kind-cluster
-# TODO restore once catalog is tested
+	kubeaddons-enterprise-tests/run-tests.sh
+	bin/kind delete cluster
```

Comment:
This seems kind of redundant if we're doing all of this in clean anyways. my suggestion is to take this out